lib/config/folderconfiguration.go

```diff
@@ -169,6 +169,13 @@ func (f *FolderConfiguration) DeviceIDs() []protocol.DeviceID {
 	return deviceIDs
 }
 
+func (f *FolderConfiguration) FSWatcherDelay() time.Duration {
+	if f.FSWatcherDelayS >= 0 {
+		return f.FSWatcherDelay() * time.Second
+	}
+	return time.Second / (-f.FSWatcherDelay())
```

Comment:
Add a large "here be dragons" comment, it's extremely confusing if you don't know that we are doing this to preserve protobuf messages